### ISSUE TITLE
How to use with RecycleView or with ScrollView?

### ISSUE THREAD

**Issue by FedericoOldrini** [kmshack/Android-ParallaxHeaderViewPager#12]
Can you post an example with ScrollView or with the new RecycleView class? Thanks

**Comment by schwiz** [comment]
So far everything I've tried with RecyclerView the Content of the RecyclerView stutters horribly as you translate the header its scroll changes by a delta of 1 or 2 in either direction.

**Comment by schwiz** [comment]
That only half way works, if the content of the scrollview isn't greater than the size of the window it looks really bad.

**Comment by FedericoOldrini** [comment]
I have also added a "footer" to the content whose height was dinamically calculated so every page can scroll at minimum the full height of the header. But is not a really  clean solution

**Comment by schwiz** [comment]
Yeah I was thinking of that, I ended up animating the header back to the original position if canScrollVertically returns false when switching the view page.  Also not really the perfect solution but functional.
